x-pack/plugins/enterprise_search/server/routes/app_search/reference_application.test.ts

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { mockDependencies, mockRequestHandler, MockRouter } from '../../__mocks__';
+
+import { registerReferenceApplicationRoutes } from './reference_application';
+
+describe('reference application routes', () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  describe('GET /api/app_search/engines/{name}/search_settings/details', () => {
```

Comment:
[not blocking] minor typo?
```suggestion
  describe('GET /api/app_search/engines/{engineName}/search_settings/details', () => {
```

not sure sure if this matters

Reply:
😅